releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -53,70 +53,116 @@
+    <choice
+        value="Oxygen - Staging"
+        label="Eclipse Oxygen - 4.7 (Staging)"/>
+    <choice
+        value="Neon"
+        label="Eclipse Neon - 4.6"/>
+    <choice
+        value="Mars"
+        label="Eclipse Mars - 4.5"/>
+    <choice
+        value="Luna"
+        label="Eclipse Luna - 4.4"/>
     <description>Choose the compatibility level of the target platform</description>
   </setupTask>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="xtext.nightly.composite"
-      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest/"/>
+      name="version.mwe2"
+      value="2.9.0"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.releases"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/releases"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.milestones"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/milestones"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.latest"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext"
+      value="${p2.xtext.latest}"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.orbit"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.mwe2"
+      value="http://download.eclipse.org/modeling/emft/mwe/updates/releases/${version.mwe2}"
+      label=""/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="mwe2.update.site"
-      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest/"/>
+      name="p2.emf"
+      value="http://download.eclipse.org/modeling/emf/emf/updates/2.12"/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="emf.update.site"
-      value="http://download.eclipse.org/modeling/emf/emf/updates/2.12/"/>
+      type="URI"
+      name="p2.gef"
+      value="http://download.eclipse.org/tools/gef/updates/releases"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      type="URI"
+      name="p2.xpand"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/xpand/R201406030414"/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="antlr-gen.update.site"
-      value="http://download.itemis.com/updates/releases/2.1.1/"/>
+      name="p2.antlr-gen"
+      value="http://download.itemis.com/updates/releases/2.1.1"/>
   <setupTask
       xsi:type="setup:VariableTask"
       type="URI"
-      name="draw2d.p2.repository"
-      value="http://download.eclipse.org/tools/gef/updates/legacy/releases/"/>
+      name="p2.draw2d"
+      value="http://download.eclipse.org/tools/gef/updates/legacy/releases"/>
   <setupTask
       xsi:type="setup:VariableTask"
       type="URI"
-      name="buildship.p2.repository"
```

Comment:
should we update to http://download.eclipse.org/buildship/updates/e47/releases/2.x ???

Reply:
I will give it a try. I was not aware of that p2 repo.